settings.gradle.kts

```diff
@@ -24,6 +24,7 @@ pluginManagement {
         mavenLocal()
         jcenter()
         maven(url = "https://dl.bintray.com/arsysop/lang")
+        maven(url = "https://dl.bintray.com/arsysop/liho")
```

Comment:
self reference?

Reply:
yah :) This repo configuration is only work when 'liho-gradle-plugin' is applied (as it requires 'liho' of release version we are working at). 

Thus, this repo ref does not breack the build and this file is not included into a release artifact.